Vagrantfile

```diff
@@ -96,4 +96,16 @@ EOF
     cent.vm.provision "shell", inline: "systemctl start docker"
     cent.vm.provision "shell", inline: "systemctl enable docker"
   end
+
+  config.vm.define "ubuntu" do |ubuntu|
+    ubuntu.vm.box = "ubuntu/xenial64"
```

Comment:
You can add this if you want, but I'm not sure it's necessary. This Vagrantfile is just to provide a dev env for testing the puppet module. It will launch containers to do the beaker tests in CentOS/Ubuntu, per your nodefile. The rspec/lint tests shouldn't change due to the host's OS.